slave/process-scim/rpm.dependencies

```diff
@@ -0,0 +1 @@
+perun-slave-base
```

Comment:
I would expect python3 here.

Reply:
The slave doesn't have dependency on python. Script written in python is just an example how to use the slave.